test/metabase/test/util_test.clj

```diff
@@ -118,3 +118,16 @@
                                    (orig))))]
       (mock-me-outer)))
   (is (= :mock/redefined (z))))
+
+(deftest ^:parallel ordered-subset?-test
```

Comment:
Out of scope, but this seems like good case for property testing. Too trivial to justify the exercise unless it's a warm-up for something more important.